_build/scripts/test.py

```diff
@@ -25,6 +25,7 @@
   parser.add_argument("-single_threaded", help="run tests in single threaded mode", action="store_true")
   parser.add_argument("-only_errors_and_warnings", help="filter lines to only display warnings and errors", action="store_true")
   parser.add_argument("-auto_fix", help="auto fix where you can (applies to iwyu and clang-tidy)", action="store_true")
+  parser.add_argument("-runs", help="num runs to perform (applies to fuzzy testing)", default=10000)
```

Comment:
I think I already commented this somewhere else but can we split up the different type of tests in a new subprocess like we do with setup/generate/build. This will solve that we have arguments for specific runs within a general place.

Reply:
you want to be able to do something like
`py _rex.py fuzz` `py _rex.py unit-test` `py _rex.py auto-test` `py _rex.py static-analysis`?
that's doable, although, probably best in another PR

Reply:
Yes that is what I want, another PR is fine

Reply:
linked to https://github.com/New-Creta/Rex/issues/272